root/user/collections.tt

```diff
@@ -15,19 +15,22 @@
 
 [% WRAPPER "user/profile/layout.tt" title=l("Collections") page="collections" %]
 
+<h2>[% l("My collections") %]</h2>
 [%- FOR colentity = collections.pairs -%]
     [% INCLUDE 'user-collections' %]
 [% END %]
 
 [%- IF no_collections -%]
-    <h2>[% l("Collections") %]</h2>
     [% IF viewing_own_profile %]
         <p>[% l('You have no collections.') -%]</p>
     [%- ELSE -%]
         <p>[% l('{user} has no public collections.', { user => html_escape(user.name) }) -%]</p>
     [%- END -%]
 [%- END -%]
 
+<h2>[% l("Collaborative collections") %]</h2>
+[% INCLUDE 'collaborator-collections' %]
```

Comment:
Maybe I don't have the terminology mapped out in my head correctly yet, but it seems slightly strange how these two lines use different terms for what I think are the same thing (collaborative vs. collaborator collections).

Reply:
So what was the resolution here? :) I'm still not sure what the difference is but they're both still used about equally.

Reply:
Collaborative sounds more English so let's use that.